test/SlowTests/Issues/RavenDB_12193.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using Xunit;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Changes;
+using Raven.Client.Extensions;
+using Raven.Tests.Core.Utils.Entities;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_12193 : RavenTestBase
+    {
+        // RavenDB-12481
+        [Fact]
+        public async Task Should_Throw_On_UnobservedTaskException()
+        {
+            var count = 0;
+
+            EventHandler<UnobservedTaskExceptionEventArgs> task = (sender, args) =>
+            {
+                Interlocked.Increment(ref count);
```

Comment:
I wonder if we could somehow check here that `sender` is the guy that was causing the issue. I'm a bit worried that we can have failures of this tests due to other unknown unobserved tasks (which might be actually fine - something in server size)

Reply:
for now it checks all UnobservedTaskException, 
 I can check if args or sender is not null if it is check if its equal to exceptions that I want to track.

Reply:
Okay, it seems we have first failure of this test:

http://jenkins.hibernatingrhinos.com/job/PR_Tests_v4.1/2223/testReport/junit/SlowTests.Issues/RavenDB_12193/Should_Throw_On_UnobservedTaskException/

It's from testing https://github.com/ravendb/ravendb/pull/8159 PR

Reply:
BTW I've just found we have: https://github.com/ravendb/ravendb/blob/436d63cd0b9b42473d7dc0945b8f4ebd2be6b2a3/test/Tests.Infrastructure/TestBase.cs#L70-L85

Regarding the above failure, personally I'm fine with skipping this test (but leave it with comment so one can run it manually if needed) if we cannot make it behave consistently. This wasn't a but that made some functionality wasn't working.

Reply:
@arekpalinski does this PR include #8159 or we still have that error there? This thing of having to run the slow tests to catch the errors get me sidetracked every day. Have like 3 PRs open that I cannot end up closing.